src/tir/op/op.cc

```diff
@@ -922,4 +922,8 @@ TVM_REGISTER_GLOBAL("tir._OpIfThenElse")
       return if_then_else(cond, true_value, false_value);
     });
 
+TVM_REGISTER_GLOBAL("tir.const_true").set_body_typed([](DataType t) {
+    return const_true(t.lanes());
+});
+
```

Comment:
Is this related with the "span" change?

Reply:
Yes, I need it in `python/tvm/tir/stmt.py` to handle passing multiple optional arguments to a C++ function (in this case the Store and Load constructors).